cli/src/spend_utils.rs

```diff
@@ -20,6 +20,8 @@ pub enum SpendAmount {
     All,
     Some(u64),
     RentExempt,
+    // will move all, but use a minimum for simulation
+    AllWithMinimum(u64),
```

Comment:
I feel like this variant name is a little unclear, how about this?
```suggestion
    AllForAccountCreation {
        create_account_min_balance: u64,
    }
```

Reply:
Yeah, that's much better, thanks!